java/cuvs-java/src/main/java/com/nvidia/cuvs/CagraIndex.java

```diff
@@ -143,13 +143,6 @@ default void serializeToHNSW(OutputStream outputStream, Path tempFile) throws Th
    */
   void serializeToHNSW(OutputStream outputStream, Path tempFile, int bufferLength) throws Throwable;
 
-  /**
-   * Gets an instance of {@link CagraIndexParams}
-   *
-   * @return an instance of {@link CagraIndexParams}
-   */
-  CagraIndexParams getCagraIndexParameters();
```

Comment:
Still looking to understand why this was removed.

Reply:
Index parameters are only using at index build time; there is no reason to retain them in the index once this is build (which, in Java, we do directly in the ctor). They are simply inputs.